packages/salesforcedx-vscode-apex/test/vscode-integration/requirements.test.ts

```diff
@@ -8,29 +8,51 @@
 // tslint:disable:no-unused-expression
 
 import { expect } from 'chai';
-import * as path from 'path';
-import * as shell from 'shelljs';
-import { workspace } from 'vscode';
-import { JAVA_HOME_KEY, JAVA_MEMORY_KEY } from '../../src/requirements';
+import { createSandbox, SinonSandbox, SinonStub } from 'sinon';
+import * as vscode from 'vscode';
+import { JAVA_HOME_KEY, resolveRequirements } from '../../src/requirements';
+import pathExists = require('path-exists');
+import * as cp from 'child_process';
+
+const runtimePath = '~/java_home/real/jdk/openjdk1.8.0.302_8.56.0.22_x64';
```

Comment:
We don't package a JDK, just enforce some versions. Do you think this should be separated out in a separate `const` file?

Reply:
Since we use `openjdk1.8.0.302_8.56.0.22_x64` in the test below - was wondering that if we reuse we can avoid having to change in 2 places when we update JDK versions. But its just a minor point.